Validation/RecoTau/plugins/DQMHistNormalizer.cc

```diff
@@ -18,15 +18,26 @@
 #include "FWCore/ServiceRegistry/interface/Service.h"
 
 //Regexp handling
-#include "classlib/utils/RegexpMatch.h"
-#include "classlib/utils/Regexp.h"
-
+#include <regex>
 #include <string>
 #include <vector>
 #include <map>
 
 using namespace std;
 
+std::string globtoregex(const std::string& s) {
+  std::string out;
+  out.reserve(s.size());
+  for (auto ch : s) {
+    if (ch == '*') {
+      out.push_back('.');
+      out.push_back('*');
+    }
+    out.push_back(ch);
+  }
+  return out;
+}
+
```

Comment:
@iarspider , why not just use `std::regexp_repalce()`  instead of this `globtoregex`?